dracut/99setup-root/initrd-setup-root

```diff
@@ -82,9 +82,12 @@ done
 /usr/sbin/flatcar-tmpfiles /sysroot
 
 # Initialize base filesystem without /etc
+# Don't run all tmpfiles rules but only the essential ones
+# required to let systemd do its work correctly. The rest
+# will be done later as usual.
 systemd-tmpfiles --root=/sysroot --create \
     baselayout.conf baselayout-usr.conf \
-    baselayout-home.conf
+    baselayout-home.conf base_image_var.conf
```

Comment:
Since you are adding `/var/log` and `/var/log/journal` in `baselayout.conf`, is adding `base_image_var.conf` here necessarry?

Reply:
It's not necessary for the topic at hand, it contains things like `/var/log/audit` and `/var/db/sudo`. I thought it's good to set things up at this point to prevent similar problems showing up again. The `base_image_var.conf` contents are generated during image build and don't originally come from tmpfiles rules but from packages. For those rules coming from tmpfiles I think it's enough to let them do the job at their regular boot stage but for those not from tmpfiles we don't really know whether this works for them, so I added it as precaution.

Reply:
Have updated the commit message.